Copy edit pass (#73)
ember-learn/super-rentals-tutorial: src/chapters/08-working-with-data.md

ember server
```

> In this chapter, we will remove the hard-coded data from our `<Rental>` component. By the end, your app should look the same, but with the ability to `fetch` this data from an API.

Reviewer: ```suggestion
In this chapter, we will remove the hard-coded data from our `<Rental>` component. By the end, your app would finally be displaying real data that came from the server!
```

It doesn't actually look the same, because the single duplicated listing will now be replaced with three "real" rental listing.

Reviewer 2: ```suggestion
In this chapter, we will remove the hard-coded data from our `<Rental>` component. By the end, your app will finally display real data that will come from the server!
```

Reviewer 2: "In this chapter, we will remove the hard-coded data from our `<Rental>` component. By the end, your app will finally display real data that comes from the server!"

Now that we have things in place, let's do the fun part of removing *all* our hard-coded values from the model hook and actually fetch some data from the server!

> In a production app, the data that we'd fetch would most likely come from a remote API server. To avoid setting up an API server just for this tutorial, we will put some JSON data into the `public` folder instead. That way, we can still request this JSON data with regular HTTP requests &mdash; just like we would with a real API server  &mdash; but without having to write any server logic.

Reviewer: There are multiple JSON files, is this change correct?

Author: Depends on whom you ask. [OED](https://www.lexico.com/en/definition/data) says "data" is a mass noun and thus gets singular "this".

[APA](https://blog.apastyle.org/apastyle/2012/07/data-is-or-data-are.html), however, says to use "these data".

Dianne and I both made this face when we read "these data": 🤔

"This data" feels less formal too.

Reviewer 3: The AP styleguide says it's singular in non-academic contexts. https://twitter.com/apstylebook/status/1111694537020370945?lang=en

That sounds about right to me.

> But where will the data come from? You can <a href="/downloads/data.zip" download="data.zip">download this data file</a>, where we have prepared some JSON data and bundled it into a `.zip` file format. Extract its content into the `public` folder.

Reviewer: Same

Author: We'll duke it out above. :-P

   }
```

> After parsing the JSON data, we extracted the nested `attributes` object, added back the missing `type` attribute manually, and then returned it from the model hook. That way, the rest of our app won't have any idea that this difference ever existed.

Reviewer 2: ```suggestion
After parsing the JSON data, we extracted the nested `attributes` object, added back the missing `type` attribute manually, and then returned it from the model hook. That way, the rest of our app will have no idea that this difference ever existed.
```
